src/engine/strat_engine/pool.rs

```diff
@@ -225,13 +237,14 @@ impl Pool for StratPool {
             }
         }
 
+        let dm = try!(DM::new());
```

Comment:
I've coopted this into #355 since it is not part of these changes, and one of my commits in #355 will conflict with it, anyway.